installers/WindowsInstaller.hs

```diff
@@ -116,8 +116,6 @@ writeInstallerNSIS fullVersion = do
     _ <- section "" [Required] $ do
         setOutPath "$INSTDIR"        -- Where to install files in this section
         writeRegStr HKLM "Software/Daedalus" "Install_Dir" "$INSTDIR" -- Used by launcher batch script
-        createDirectory "$APPDATA\\Daedalus\\DB-0.2"
```

Comment:
It's a minor unrelated bug Ilya and I noticed where we're unnecessarily creating directories for old versions. I addressed it in its [own commit](https://github.com/input-output-hk/daedalus/pull/316/commits/372adc9a044f9c90be8a453e9648432f59b93c56).